packages/webview_flutter/webview_flutter_android/test/android_webview_controller_test.mocks.dart

```diff
@@ -747,11 +776,13 @@ class MockAndroidWebViewProxy extends _i1.Mock
     void Function(
       _i2.WebView,
       int,
-    )? onProgressChanged,
+    )?
```

Comment:
it might be a setting on your IDE

Reply:
It's because of this command `dart pub global run flutter_plugin_tools format`